manifests/config/passenger.pp

```diff
@@ -51,7 +51,13 @@
   $prestart            = $::foreman::passenger_prestart,
   $min_instances       = $::foreman::passenger_min_instances,
   $start_timeout       = $::foreman::passenger_start_timeout,
+  $infrastructure_port = 9070,
+  $client_ca      = $::foreman::client_ssl_ca,
 ) {
+
+  $infrastructure_url = "https://${servername}:${infrastructure_port}"
+  $infrastructure_redirection_path = "/"
```

Comment:
I'd put these just above the vhost definition since thats where they are used (localizes them for context).